web/l10n_utils.js

```diff
@@ -24,12 +24,12 @@
  * A subset of the l10n strings in the `l10n/en-US/viewer.properties` file.
  */
 const DEFAULT_L10N_STRINGS = {
-  of_pages: "of {{pagesCount}}",
-  page_of_pages: "({{pageNumber}} of {{pagesCount}})",
+  of_pages: "of { $pagesCount }",
```

Comment:
*This applies everywhere below as well, and ties into my (previous) question about the general state of `NullL10n` class.*

Given that the `.ftl` file calls this `of-pages`, I don't think this'll work?